src/main/scala/BenchmarkUtils.scala

```diff
@@ -94,6 +94,28 @@ trait BenchmarkUtils {
 
   lazy val singleThreadedMonixScheduler: monix.execution.Scheduler =
     monix.execution.Scheduler(BenchmarkUtils.singleThreadedExecutor)
+
+  // Default scheduler for Monix-related benchmarks
+  implicit lazy val monixScheduler: monix.execution.Scheduler = {
+    import monix.execution.Scheduler
+    import monix.execution.ExecutionModel.SynchronousExecution
+    import monix.forkJoin.ForkJoinExecutionContext
+
+    // Building a Scala 2.11 ForkJoinPool, because the one in
+    // Java 8 / Scala 2.12 has worse performance due to being more fair;
+    // See: https://github.com/monix/monix-forkjoin/
+    val executionContext = 
```

Comment:
Since Monix, FS2, and Scalaz-Stream all create a Scheduler or Strategy from an ExecutionContext, I wonder whether it makes sense to extract this out, and then use the same for all three.  That would help address the concern in #3.

Whether that should be a fixed thread pool or a fork join pool is a separate question.  Both, on all three libraries, could be interesting.